client/app/bundles/HomeComponent/components/HomeComponent.jsx

```diff
@@ -1,13 +1,27 @@
-import React, { PropTypes } from 'react';
+import React from 'react';
+import BasicStatInput from '../../BasicStatInput/components/BasicStatInput';
+import LocaleSwitcher from '../../LocaleSwitcher/components/LocaleSwitcher';
+import Home from '../../Translations/components/Home';
 
 export default class HomeComponent extends React.Component {
+  selectFieldChange = (event) => {
```

Comment:
Consider renaming to `selectFieldChanged` or `handleSelectFieldChange` or even `handleCharacterClassChange`